client/state/selectors/get-primary-domain-by-site-id.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal Dependencies
+ */
+import { getDomainsBySiteId } from 'state/sites/domains/selectors';
+
+/**
+ * Return primary domain from state object and
+ * the given site ID
+ *
+ * @param {Object} state - current state object
+ * @param {Object} siteId - site object
+ * @return {Object} primary domain
+ */
+export default function getPrimaryDomainBySiteId( state, siteId ) {
+	const domains = getDomainsBySiteId( state, siteId );
+	if ( domains.length === 0 ) {
+		return null;
+	}
+
+	return domains.filter( domain => domain.isPrimary )[ 0 ];
```

Comment:
[`find` seems a better fit](https://lodash.com/docs#find), if you're only interested in the first result.

Reply:
Wanted to use the native function instead of `lodash`.

Reply:
Cool, but why? I'm guessing lodash is probably "plaguing" our bundles already ;) (or is there a rule against using lodash in selectors?)